RecoJets/JetProducers/python/ecfAdder_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+ECF = cms.EDProducer("ECFAdder",
```

Comment:
I go with ECF to be synchronized with Njettiness. In this way the (potenial) userdata name in MiniAOD is simpler.